src/Dtos.cs

```diff
@@ -10,6 +10,11 @@ public interface IFrontMatter
     /// Useful for generating tag pages.
     /// </summary>
     List<string> Tags { get; set; }
+    
+    /// <summary>
+    /// If true, the blog post will not be generated.
+    /// </summary>
+    bool Draft { get; set; }
```

Comment:
I am suggesting:
- Name it `IsDraft`, which strongly implies it's a bool.
- Create default implementation: `bool IsDraft => false;`, which doesn't force user to implement IsDraft in their FrontMatter classes..

Reply:
Done. Both excellent suggestions. Thank you sir.